src/promptflow/promptflow/_core/openai_injector.py

```diff
@@ -13,9 +13,9 @@
 import openai
 
 from promptflow._core.operation_context import OperationContext
-from promptflow.contracts.trace import TraceType
+from promptflow.tracing._trace import _traced_async, _traced_sync
```

Comment:
we should also move this file to promptflow.tracing?

Reply:
Let's do it in another PR